src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -103,7 +103,8 @@ internal partial class CodeFixService : ICodeFixService
                 document, range, GetShouldIncludeDiagnosticPredicate(document, priority),
                 includeSuppressedDiagnostics: false, priority, cancellationToken).ConfigureAwait(false);
 
-            var spanToDiagnostics = ConvertToMap(allDiagnostics);
+            var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+            var spanToDiagnostics = ConvertToMap(text, allDiagnostics);
```

Comment:
so thsi is the codefix service, asking about the most severe fix for a document.  That happens with a lightbulb request for an open document.  So getting teh text here shoudl be free.